### DISCUSSION TITLE
Imagenes logo farmaceutica

### DISCUSSION THREAD

**Discussion by ArmandoRuiz13** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#328] [General]
en que pagina podria sacar esas imagenes, alguien tiene una pagina para imagenes

**Comment by SujeiRodriguez** [comment]
ah de los logos perdon XD

**Reply by ArmandoRuiz13** [reply]
Si, gracias igual

**Comment by CesarMezaB** [comment]
En Freepik puedes encontrar:
https://www.freepik.com/free-photo/front-view-female-nurse-white-medical-suit-with-red-mask-injection-her-hands-blue_13935723.htm?query=nurses%20%20nursing#position=30&query=nurses++nursing

**Reply by SujeiRodriguez** [reply]
oh no, ud también ha caído en la trampa... XD
dice de los nombres de las farmacéuticas

a ver, yo hice un recorte de la imagen, pero aun no la pongo, supongo que se vera bien
la otra es buscar cada una como "logo astrazeneca" y tratarlos con photoshop

**Reply by OmarVenturaP** [reply]
Yo lo hice así, cada logo lo fui buscando y los inserte. Si es un poco de trabajo, pero pues al fin de cuentas se ve bien. 😁

**Reply by SujeiRodriguez** [reply]
XD yo así le iba a hacer con los iconos que no encontrara, pero ya vi que se pueden importar

**Comment by DanyVeneno** [comment]
Que tal Armando
pexels.com es una
